test/base/ProfilerTest.ooc

```diff
@@ -0,0 +1,33 @@
+use ooc-base
+use ooc-unit
+import io/File
+
+ProfilerTest: class extends Fixture {
+	init: func {
+		super("Profiler")
+		this add("log", func {
+			outputFile := "test/test.log"
```

Comment:
Also, I noticed in another test that you created strings as member variables and freed them in a `free` function. Any reason not to do that here?